msvc/config.h

```diff
@@ -0,0 +1,4 @@
+#define HAVE_AVX2   1
+#define HAVE_NEON32 0
+#define HAVE_NEON64 0
+#define HAVE_SSSE3  1
```

Comment:
Maybe it could be generated with something like bootstrap.bat.
But MSVC requires different kind of projects for ARM (different platform toolsets)
E.g. Windows Phone 8.1 Static Library, DLL, Windows App Library, Universal Apps library,...
I'm actually not a mobile developer to correctly deal with it.
So I just created a simple project for "old-fashioned" x86 platform.